### ISSUE TITLE
Remove the title caption whilst maintaining accessibility of the charts

### ISSUE THREAD

**Issue by christiemolloy** [RedHatInsights/insights-dashboard#150]
A usability issue was discovered in the dashboard. When hovering over any of the charts a caption for the chart will appear after hovering over the chart for a number of seconds. This can block the label for the chart if the position of the mouse if close to where the label sits.

The reason for this caption is because Victory charts uses a `<title>` for accessibility purposes to label the chart, instead of using something like `aria-label`. As PatternFly Charts is a wrapper around Victory, we cannot inject an `aria-label` into the `<svg>` however we have the ability to not pass the `title` prop through to the chart.

We would need to test different solutions here like:
- using just the `description` prop, as this will not render a caption like title does, and this might be enough to still accurately describe the chart
- wrap the chart in an aria-label, however this will be problematic because then the `svg` doesn't directly have a label/caption
- create an issue with Victory that would allow us to pass a prop into the chart as the `role` and/or pass in an aria-label for the svg to replace the title.

![](https://user-images.githubusercontent.com/28281340/80411554-5f187180-889a-11ea-977f-95df097f493e.png)
